gui/widgets/popup.cpp

```diff
@@ -510,7 +535,11 @@ void PopUpWidget::drawWidget() {
 	Common::String sel;
 	if (_selectedItem >= 0)
 		sel = _entries[_selectedItem].name;
-	g_gui.theme()->drawPopUpWidget(Common::Rect(_x, _y, _x + _w, _y + _h), sel, _leftPadding, _state);
+
+	if (g_gui.useRTL() && _useRTL)
+		_leftPadding = 0;
```

Comment:
Maybe it should be something like `padding = _rightPadding` for RTL and  `padding = _leftPadding` for LTR?

Reply:
This ties back to the `x += deltax;` in `graphics/font.cpp`.
I have to put paddings as 0, otherwise they will go out of bound.

In order to make these work, I would have to reintroduce the thing earlier with
`if (align == kTextAlignRight) then x -= deltax`

I know you gave approval on this, but all the last char issues are now fixed. If I were to enable the above code, it would allow me to add some padding. I thought it would be best if I asked again.

Reply:
Fixed now, all right aligns now have a neat little padding :)